Subject: [PATCH] healthd: Use project pathmap for recovery

diff --git a/healthd/Android.mk b/healthd/Android.mk
index 8b59964..0e685a7 100644
--- a/healthd/Android.mk
+++ b/healthd/Android.mk
@@ -46,7 +46,7 @@
     AnimationParser.cpp
 
 LOCAL_MODULE := libhealthd_charger
-LOCAL_C_INCLUDES := bootable/recovery $(LOCAL_PATH)/include
+LOCAL_C_INCLUDES := $(call project-path-for,recovery) $(LOCAL_PATH)/include
 LOCAL_EXPORT_C_INCLUDE_DIRS := \
     $(LOCAL_PATH) \
     $(LOCAL_PATH)/include
