diff --git a/rootdir/init.rc b/rootdir/init.rc
index b4a25fe..a0397a0 100644
--- a/rootdir/init.rc
+++ b/rootdir/init.rc
@@ -10,6 +10,9 @@
 import /vendor/etc/init/hw/init.${ro.hardware}.rc
 import /init.usb.configfs.rc
 import /init.${ro.zygote}.rc
+import /init.kernel.rc
+# Include lineage extra init file
+import /init.lineage.rc
 
 on early-init
     # Set init and its forked children's oom_adj.
